File: examples/functions/assets-static-first/node/api-node/handler.js
exports.handler = async () => ({
  status: 200,
  headers: { "Content-Type": "application/json" },
  body: JSON.stringify({
    runtime: "node",
    title: "Node fallback route",
    summary: "Static assets own / first, but /api-node still resolves to the Node handler.",
    path: "/api-node",
  }),
});

File: examples/functions/assets-static-first/python/api-python/handler.py
import json


def handler(_event):
    return {
        "status": 200,
        "headers": {"Content-Type": "application/json"},
        "body": json.dumps(
            {
                "runtime": "python",
                "title": "Python fallback route",
                "summary": "The gateway misses on public/ and then dispatches to Python.",
                "path": "/api-python",
            },
            separators=(",", ":"),
        ),
    }
